INSTALL.md

```diff
@@ -0,0 +1,121 @@
+```SQL
+CREATE USER telemetry WITH PASSWORD '$PG_PASS_TELEMETRY';
+CREATE USER grafana WITH PASSWORD '$PG_PASS_GRAFANA';
+CREATE USER grafana_ro WITH PASSWORD '$PG_PASS_GRAFANA_RO';
+CREATE USER dashboard WITH PASSWORD '$PG_PASS_DASHBOARD' NOINHERIT;
+CREATE DATABASE telemetry OWNER telemetry;
+```
+
+#### Import DDLs
+```bash
+cd ~/ceph-telemetry
+psql -v ON_ERROR_STOP=1 -b -h 127.0.0.1 -U telemetry telemetry < tables.txt
+psql -v ON_ERROR_STOP=1 -b -h 127.0.0.1 -U postgres telemetry < db_create_cluster.sql
+psql -v ON_ERROR_STOP=1 -b -h 127.0.0.1 -U telemetry telemetry < db_create_device.sql
+psql -v ON_ERROR_STOP=1 -b -h 127.0.0.1 -U postgres telemetry < db_create_roles.sql
+psql -v ON_ERROR_STOP=1 -b -h 127.0.0.1 -U grafana telemetry < db_create_dashboard.sql
+psql -v ON_ERROR_STOP=1 -b -h 127.0.0.1 -U grafana telemetry < db_create_dashboard_device.sql
+```
+### Configure Grafana
+1. Login to Grafana via a browser (port 3000) with the default username 'admin' and password 'admin'.
+2. Configure a data source of the postgres server
+   1. Use `grafana_ro` as the database user, with the password that is saved in `/opt/telemetry/pg_pass_grafana_ro`
+   2. You may need to use the host's IP address (not localhost)
+
+## Install Apache HTTP server
+1. run:
+```bash
+sudo dnf install -y httpd python3-mod_wsgi mod_ssl mod_evasive openssl python3-requests python3-flask python3-flask-restful python3-psycopg2 lz4
+# sudo systemctl enable --now httpd
+sudo cp ~/ceph-telemetry/install/telemetry-ssl.conf /etc/httpd/conf.d/
+```
+2. Generate web server certificates for the telemetry server's public FQDN. Below instructions are of how to generate self-signed certificates that should not be used in production
+```bash
+sudo mkdir -p /etc/telemetry/ssl
+sudo openssl req -x509 -nodes -newkey rsa:2048 -keyout /etc/telemetry/ssl/telemetry.key -out /etc/telemetry/ssl/telemetry.crt
+```
+3. Edit `/etc/httpd/conf.d/telemetry-ssl.conf` and change the following to match your environment:
+   - ServerName
+   - SSLCertificateFile, SSLCertificateKeyFile
+4. You may need to configure SELinux to allow httpd access to the telemetry wsgi using `semanage permissive -a httpd_t`
+
+## Install the Telemetry server
+run:
+```bash
+cd ~/ceph-telemetry
+sudo cp -a server /opt/telemetry/
+sudo cp import_crashes.py import_clusters.py import_devices.py compress_raw_reports_telemetry.sh dbhelper.py compress_raw_reports_telemetry.sh /opt/telemetry/
+cd /opt/telemetry
+sudo ln -s pg_pass_telemetry pg_pass.txt
+sudo mkdir log
+sudo chown apache log
+sudo mkdir raw
+sudo chmod a+rwx raw
+```
+#### Add Telemetry importers to cron
+Create a "telemetry" user unix account and then run:
+```bash
+sudo crontab -u telemetry install/crontab_telemetry
+```
+
+# Backing up the Telemetry server
+Backing up the server involves backing up the PostgreSQL database by running
+```bash
+PGPASSWORD=<POSTGRESS PASSWORD> pg_dumpall postgres | lz4 -c" > telemetry_server.sql.lz4

```

Comment:
`<POSTGRESS PASSWORD> => postgres_password` (same case as above),  Maybe also mention "same as in the docker-compose.yml file"

Reply:
fixed